### DISCUSSION TITLE
Stop due to high data traffic

### DISCUSSION THREAD

**Discussion by datemitumasa** [HumanSupportRobot/QandA#21] [Q&A] [answered]
When subscribing depth image and rgb image with HSR, the warn "**Cycle over rate exceeds 0.1**." occurs. If the number of topic & data to be communicated is increased while leaving this warn unattended, HSR will be stopped suddenly.

Is anyone else experiencing a similar warn?
Or does anyone know of this warn?

**Comment by hsr-admin** [comment, marked as answer]
Reducing the CPU load may improve it.

Example.)
　Disable the marker recognition function by referring to the following manual
　https://docs.hsr.io/hsr_develop_manual_en/customize/sensor_on_off.html

　Disable sensors, etc. referring to the following manual
　https://docs.hsr.io/hsr_develop_manual_en/customize/sensor_on_off.html#id1
